### ISSUE TITLE
[Bug] Could not be loaded in Wireshark 4.4.2

### ISSUE THREAD

**Issue by kaolalong** [mauricelam/btsnoop-extcap#4]
System:
Ubuntu 24.04
Steps:
I use the installation method to install this into Wireshark 4.4.2, which is built by myself from source code.
Result:
Check "help->About Wireshark->Plugins", there is no "btsnoop-extcap" shown in the list.
Open Wireshark and there is no related interface shown also.

**Comment by mauricelam** [comment]
Is `adb` available on your path? Looks like I am missing a few error checks for this case and it could be causing the program to abort too early.

**Comment by kaolalong** [comment]
@mauricelam Thanks for the reply.  I debugged this issue.  Finally, I found the reason is how the .desktop file to exec `wireshark` command.  The original command is:
```
 Exec=wireshark %f
```
With this command, all my personal env variables are not loaded.  Just like you said, the `adb` path was not set also.
I changed the command as below:
```
Exec=bash -i -c "wireshark %f"
```
All my env variables were loaded.
Not sure if this is the gnome or wireshark issue.  But definitely not your extcap issue.
Thanks.
